terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -10,13 +10,14 @@ resource "aws_iam_account_alias" "alias" {
 }
 
 module "cross-account-access" {
-  source = "github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access?ref=v2.1.1"
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access?ref=v2.2.0"
   providers = {
     aws = aws.workspace
   }
-  account_id = local.modernisation_platform_account.id
-  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
-  role_name  = "ModernisationPlatformAccess"
+  account_id             = local.modernisation_platform_account.id
+  additional_trust_roles = [format("arn:aws:iam::%s:role/github-actions", local.environment_management.account_ids[terraform.workspace])]
+  policy_arn             = "arn:aws:iam::aws:policy/AdministratorAccess"
+  role_name              = "ModernisationPlatformAccess"
```

Comment:
Code looks good but the member account terraform assumes the `MemberInfrastructureAccess` so it should go on that role :)